docs/modules/ROOT/pages/usage/lsp.adoc

```diff
@@ -125,10 +125,17 @@ https://github.com/helix-editor/helix[Helix] is a post-modern modal text editor
 Add the following to your Helix language configuration file (e.g. `~/.config/helix/languages.toml`):
 
 ```toml
+[language-server.rubocop]
+command = "bundle"
+args = ["exec", "rubocop", "--lsp"]
+
 [[language]]
 name = "ruby"
-language-server = { command = "bundle", args = ["exec", "rubocop", "--lsp"] }
 auto-format = true
+language-servers = [
+  { name = "rubocop" }
+]
```

Comment:
Is Helix 2.3.10 compatible with earlier versions? If not, it might be good to show examples of the TOML for each.

Reply:
This is not a backward-compatible change. I pushed a new commit to keep the original example using the old structure, which can eventually be removed.